insights/__init__.py

```diff
@@ -14,7 +14,8 @@
 from .core.plugins import make_response, make_metadata  # noqa: F401
 from .core.filters import add_filter, apply_filters, get_filters  # noqa: F401
 from .parsers import get_active_lines  # noqa: F401
-from .util import defaults, parse_table  # noqa: F401
+from .util import defaults  # noqa: F401
+from .parsers import parse_delimited_table as parse_table  # noqa: F401
```

Comment:
In the `master` branch, do you think it's still necessary to keep the old `parse_table` here? how about use the `parse_delimited_table` directly?
And would you mind put the `parse_fixed_table` here too?

Reply:
Well, we export neither parse_delimited_table nor parse_fixed_table, so IMO that's the way it should be.  The purpose of this move and rename was to move the `parse_delimited_table` into the `parsers` module where it is commonly used.  IMO we'd like to avoid other things using these internal functions to parse text, and would instead like them to use the parsers :-)

So thanks for reminding me to not export the symbol I didn't want to export :-)

Reply:
It's OK to me that do not export them there.